views/editor/bramble.html

```diff
@@ -117,10 +117,13 @@
 
     <div id="spinner-container">
         <div class="thimble-spinner"></div>
-        <div class="loading-message message">{{ gettext("loadingProject") }}</div>
+        <div class="loading-message message">{{ gettext("loadingProject") }}
+	<p>{{ gettext("loading problem") }} <button type="button"><a href="https://support.mozilla.org/products/webmaker/thimble">{{ gettext("get help here") }}</button></a></p>
```

Comment:
Since there will be html in the localized string, your `gettext` query will also change a bit. You will need to write something like `{{ gettext("whateverKeyYouDecide") | safe }}`